recipes-openxt/libxcdbus/libxcdbus_git.bb

```diff
@@ -1,16 +1,16 @@
 DESCRIPTION = "XenClient DBUS library"
 LICENSE = "LGPLv2.1"
 LIC_FILES_CHKSUM = "file://COPYING;md5=321bf41f280cf805086dd5a720b37785"
-DEPENDS = "dbus dbus-glib xenclient-idl xenclient-rpcgen-native libtool libevent"
+DEPENDS = " \
+    libtool \
+    libevent \
+"
 
 PV = "0+git${SRCPV}"
 
 SRCREV = "${AUTOREV}"
 SRC_URI = "git://${OPENXT_GIT_MIRROR}/libxcdbus.git;protocol=${OPENXT_GIT_PROTOCOL};branch=${OPENXT_BRANCH}"
 
-EXTRA_OECONF += "--with-idldir=${STAGING_IDLDIR}"
-
 S = "${WORKDIR}/git"
 
-inherit autotools-brokensep pkgconfig lib_package xenclient
-
+inherit autotools-brokensep pkgconfig lib_package xc-rpcgen-c-1.0
```

Comment:
Why remove `inherit xenclient` here but not in libicbinn?

Reply:
I tend to remove it as I go and apparently missed it for libicbinn recipe. I will amend that.
The only thing provided by `xenclient.bbclass` is to cripple `do_rm_work()`, which should be done differently, and is not done by default anyway.

Reply:
Actually I seem to have left it in general... So removing it should probably be another PR all together.